modules/thermal_hydraulics/test/tests/components/volume_junction_1phase/tests

```diff
@@ -64,6 +64,16 @@
     ad_indexing_type = 'global'
   []
 
+  [calorically_imperfect_gas]
+    type = 'CSVDiff'
+    input = 'junction_with_calorifically_imperfect_gas.i'
+    csvdiff = 'junction_with_calorifically_imperfect_gas_out.csv'
+    abs_zero = 2e-10
+    rel_err = 0
+    max_parallel = 1
```

Comment:
@joshuahansel , do we need that restriction?

Reply:
I don't think we should need it, so that should be removed (and then we'll find out for sure!).

Reply:
It works but convergence seems significantly degraded (runtime increased from 1.6 to 12s on my local machine with a -devel executable)